internal/core/api.rs

```diff
@@ -121,6 +121,39 @@ impl Default for CloseRequestResponse {
     }
 }
 
+pub use crate::input::PointerEventButton;
+
+/// A mouse or touch event
+#[derive(Debug, Clone, Copy, PartialEq)]
+#[allow(missing_docs)]
+pub enum PointerEvent {
+    /// The mouse or finger was pressed
+    Pressed { position: euclid::Point2D<f32, LogicalPx>, button: PointerEventButton },
+    /// The mouse or finger was released
+    Released { position: euclid::Point2D<f32, LogicalPx>, button: PointerEventButton },
+    /// The position of the pointer has changed
+    Moved { position: euclid::Point2D<f32, LogicalPx> },
+    /// Wheel was operated.
+    /// `pos` is the position of the mouse when the event happens.
+    /// `delta` is the amount of pixel to scroll.
+    Wheel { position: euclid::Point2D<f32, LogicalPx>, delta: euclid::Vector2D<f32, LogicalPx> },
+    /// The mouse exited the item or component
+    Exit,
```

Comment:
I think that perhaps this should be called `Exited`, for consistency with `Pressed`, `Released` and `Moved`. What do you think?

Reply:
We could also rename the other to  Up, Down, Move  as i've seen in other framework

Reply:
I’m not in favor of that, for various reasons - including the fact that we use the term pressed in our existing Slint API.